README.md

```diff
@@ -101,7 +101,6 @@ X-Openstack-Request-Id: txbb13222da0d94527a7101-00649551fc
 - [ ] Refactor conf file copying, Secret and ConfigMap usage
 - [ ] Use Kolla
 - [ ] Support Network isolation
```

Comment:
Should we remove `Support Network isolation` from this list? I think we're now able to pass `net-attach-def` to the `Swift` services, so we should be good, unless I'm overlooking any extra work we need or tests we might want to include

Reply:
I'll fix that in a separate PR - noticed there is another minor NAD issue.

Reply:
Sounds good